addons/account/tests/test_account_move_out_invoice.py

```diff
@@ -2237,7 +2237,7 @@ def test_out_invoice_post_2(self):
             **self.move_vals,
             'payment_reference': move.name,
             'currency_id': self.currency_data['currency'].id,
-            'date': fields.Date.from_string('2017-01-01'),
+            'date': fields.Date.today(),
```

Comment:
You should avoid "real today" in tests to avoid not determinist error in runbot. Use `@freeze_time(...)` to the test plz

Reply:
Did I do it right? If so, it is done.